### ISSUE TITLE
Please update the docs

### ISSUE THREAD

**Issue by flayks** [agentejo/cockpit#890]
What about the docs? I updated my install to 0.8.2 and now the Regions are not there anymore, took me a while to see that Singletons are now in place (and had to remake all my pages again).
Could you update the doc for solid informations? It's very hard to follow up with the new features & co.
Thank you 🙏

**Comment by mpartipilo** [comment]
Felix, if you had asked earlier someone could have told you that regions
are now out of core and became a plugin that you can still install
separately.

That would have saved you the time to convert everything to a singleton.

**Comment by flayks** [comment]
@mpartipilo I haven't updated my site for months so I can't go look by myself for all the big changes that happened on the CMS, that's usually why docs are there IMHO.

**Comment by nilsborg** [comment]
Gotta agree with @flayks here … Docs is an annoying but equally important topic

**Comment by mpartipilo** [comment]
Sure, docs need to be kept up to date. I'm just saying that I would have asked something as soon as I saw the regions disappear :smile:

Docs are open source. Take a moment to update the bits you think need some work and do a PR. I did it for the Forms module. I will do it for singletons and regions if I find some time.

**Comment by jankal** [comment]
@mpartipilo @flayks @nilsborg We are currently working on new docs for Cockpit. For an early in-development version head [here](https://zeraton.gitlab.io/cockpit-docs/)
